lib/hydra/works/services/virus_checker_service.rb

```diff
@@ -0,0 +1,73 @@
+module Hydra::Works
+  # Responsible for checking if the given file is a virus. Coordinates
+  # with the underlying system virus scanner.
+  class VirusCheckerService
+    attr_accessor :original_file, :system_virus_scanner
+
+    # @api public
+    # @param original_file [String, #path]
+    # @return true if no virus is detected
+    # @return false if the virus scanner was not configured
+    # @return false if a virus was detected
+    def self.verify_file_is_not_a_virus?(original_file)
```

Comment:
I dislike negatives but was aiming for a similar return value. However, since it is an isolated class, I like your suggestion.